src/components/molecules/UiAlert/UiAlert.vue

```diff
@@ -67,6 +74,10 @@ const props = withDefaults(defineProps<AlertProps>(), {
   hasIcon: true,
   iconAlertAttrs: () => ({}),
   textMessageAttrs: () => ({}),
+  alertAttrs: () => ({
```

Comment:
this name is inconsistent with the convention Attrs be used to deepest passing props like on top `textMessageAttrs`

Reply:
@pspaczek, I don't understand the second sentence. 
Could you rephrase it to be more grammatically correct, please?

Reply:
...Attrs in CL is reserved pattern https://component.infermedica.com/?path=/docs/getting-started-development-guide-template-configuration--docs#attrs-props

Reply:
Okay, thanks for clarification! ❤️